ui/src/components/imquery/addNewFeatureDialog/PathDisplay.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+  <div class="flex align-items-baseline">
+    <Button v-if="canClearPath" @click="emit('onClearPath')" class="builder-button" :severity="'danger'" icon="fa-solid fa-x" text />
+    <div v-if="path.path">{{ path.path?.[0].name }} -> {{ path.typeOf?.name }} . {{ path.where?.[0]?.name }}</div>
```

Comment:
what does it show if no typeof/where? Does it need default values to avoid undefined? Or change logic to v-if